packages/reports/addon/components/filter-values/dimension-date-range.ts

```diff
@@ -44,9 +41,9 @@ export default class DimensionDateRange extends Component {
    * @param {String} value - first value to be set in filter
    */
   @action
-  setLowValue(value) {
-    this.onUpdateFilter({
-      values: [Moment(value).format('YYYY-MM-DD'), get(this, 'filter.values.lastObject')]
+  setLowValue(value: Moment.MomentInput) {
```

Comment:
Is this a string (the comment block indicates a string)?

Reply:
Doesnt seem to make a difference in functionality, switch the ts type to string